openshift/buildConfig-template.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: thoth-package-releases-buildconfig
+  thoth: 0.1.0
+metadata:
+  name: thoth-package-releases-buildconfig
+  annotations:
+    description: This is Thoth Core - Package Releases Job BuildConfig
+    openshift.io/display-name: "Thoth: Package Releases Job BuildConfig"
+    version: 0.1.0
+    tags: poc,thoth,ai-stacks,package-releases
+    template.openshift.io/documentation-url: https://github.com/Thoth-Station/
+    template.openshift.io/long-description: This template defines resources needed to deploy Thoth Package Releases Job as a Proof-of-Concept to OpenShift.
+    template.openshift.io/provider-display-name: Red Hat, Inc.
+
+objects:
+- kind: BuildConfig
+  apiVersion: v1
+  metadata:
+    name: package-releases
+    namespace: ${THOTH_BACKEND_NAMESPACE}
```

Comment:
Not sure if we want to use backend namespace for this as backend was meant to serve async requests/recommendations for end-users. Probably frontend is better place to put this job in.

Reply:
So frontend is purely for application operation overhead itself, middletier is a pool for analyzing, backend is for calculating recommendations.